cmd/manifest/manifest.go

```diff
@@ -111,38 +109,32 @@ func getManifestSource(ctx context.Context, clients *shared.ClientFactory, cmd *
 		}
 		return config.ManifestSourceRemote, nil
 	}
-	if clients.Config.WithExperimentOn(experiment.BoltFrameworks) {
-		if !cmd.Flag(manifestFlagSource).Changed {
-			manifestConfigSource, err := clients.Config.ProjectConfig.GetManifestSource(ctx)
-			if err != nil {
-				return "", err
-			}
-			switch {
-			case manifestConfigSource.Equals(config.ManifestSourceLocal):
-				return manifestConfigSource, nil
-			case manifestConfigSource.Equals(config.ManifestSourceRemote):
-				return "", slackerror.New(slackerror.ErrInvalidManifestSource).
-					WithMessage(`Cannot get manifest info from the "%s" source`, config.ManifestSourceRemote).
-					WithRemediation("%s", strings.Join([]string{
-						fmt.Sprintf("Find the current manifest on app settings: %s", style.LinkText("https://api.slack.com/apps")),
-						fmt.Sprintf("Set \"manifest.source\" to \"%s\" in \"%s\" to continue", config.ManifestSourceLocal, filepath.Join(".slack", "config.json")),
-						fmt.Sprintf("Read about manifest sourcing with %s", style.Commandf("manifest info --help", false)),
-					}, "\n"))
```

Comment:
📣 These error details are never reached due to the `err` from `GetManifestSource` above - opting to fallback to the default message if this errors!